drivers/flash/flash_stm32_qspi.c

```diff
@@ -514,7 +760,11 @@ void HAL_QSPI_TxCpltCallback(QSPI_HandleTypeDef *hqspi)
 /*
  * Status Match callback.
  */
-void HAL_QSPI_StatusMatchCallback(QSPI_HandleTypeDef *hqspi)
+#ifndef STM32_OSPI_MIMIC_QSPI
+void HAL_QSPI_StatusMatchCallback(QSPI_HandleTypeDef * hqspi)
+#else
+void HAL_OSPI_StatusMatchCallback(OSPI_HandleTypeDef *hqspi)
```

Comment:
Since these methods are defined in a header file, they shouldn't be renamed right?

Reply:
These functions are defined in the HAL libraries that Zephyr pulls in.